resources/templates/md/posts/2016-04-04-state-management.md

```diff
@@ -0,0 +1,364 @@
+{:title "Clojure state management"
+:description  "How we can manage states in Clojure (how data might change over time), or llamas love candies too :)"
```

Comment:
State, without "s"... but I would write a longer description and gently introduce to the problem... maybe already here hint at immutability and the separation between identities and values, just to make the user curious